### DISCUSSION TITLE
ESP32 local connection write-up?

### DISCUSSION THREAD

**Discussion by kongjudas** [sockless-coding/panasonic_cc#335] [General]
I'm probably out of line discussing this here, but this is where the proper nerds hang. Could someone please post links to the parts needed to make an ESP32 connection to get rid of the cloud and Panasonic as a company? They refuse to enter the 2020's and I'm bored with it. As I understand it, we need: 

ESP32
5 --> 3.3 volt 
Connector
Software
ESP Home
 
Links on Aliexpress would be lovely!

**Comment by fursov** [comment]
Do you mean something like this?
https://www.espthings.io/index.php/2023/09/02/esphome-panasonic-climate-interface/

**Reply by kongjudas** [reply]
I know that one, but it's way too complicated for most people. Just ordering the PCB is way over my head on the recommended site.

**Reply by fursov** [reply]
did you also followed this thread? https://github.com/DomiStyle/esphome-panasonic-ac/issues/111#issue-2118880716

I think, people were trying to use regular ESPs for this purpose without need to order any PCB. Also recent ESPs support 5v, no need to have any converters.

**Reply by kongjudas** [reply]
That's exactly what I was looking for! Thank you 👍 Now, I'm probably about to trash my AC with my ten thumbs. Wish me luck!
